src/views/MaterialTopTabBar.js

```diff
@@ -3,11 +3,11 @@
 import * as React from 'react';
 import { View, StyleSheet } from 'react-native';
 import { TabBar } from 'react-native-tab-view';
-import Animated from 'react-native-reanimated';
 import type {
   ViewStyleProp,
   TextStyleProp,
 } from 'react-native/Libraries/StyleSheet/StyleSheet';
+import Animated from '../modules/Animated';
```

Comment:
Re-adding comment since the previous one is gone. Material tab bar uses reanimated specific APIs, so this is not enough. Perhaps we want to provide a stub for material tabs for now?